CMakeLists.txt

```diff
@@ -478,11 +478,6 @@ else()
     endif(NOT HAVE_STRTOK_R)
 endif(WIN32)
 
-if(HAVE_REMOTE)
```

Comment:
Please submit this as a separate pull request.

Reply:
So why is the remote capture stuff moved that late in the file? I view the remote capture client part, at least, as being just another capture type, so it belongs with them.

In a511023dd7ebacff23ab220aa8315ccff2597a3b I've moved it right after the TurboCap tests in both autoconf and CMake.

Reply:
mostly to get all needed libraries into PCAP_LINK_LIBRARIES, which this early doesn't include any.
On windows, those would be winsock 2 and packet.

Reply:
update: the new place is fine. I will revert this change.